website/docs/contributing/general/ADR/domain-language.md

```diff
@@ -0,0 +1,34 @@
+---
+title: "ADR: Domain language"
+---
+
+## Background
+
+In the codebase, we have seen the need to define a domain language that we use to refer to features, methods to keep it consistent across the codebase. This ADR will contain a growing list of domain language used to keep the consistency across the codebase.
+
+## Decision
+
+We have decided to use the same domain language for the features we develop. Each feature will have it's own domain language to keep it consistent across the codebase.
+
+## Change requests domain language
+
+* *Change request*
+
+An entity referring to the overarching data structure of a change request. A change request contains changes, and can be approved or rejected.
```

Comment:
Alternative Suggestion "A  change request contains a collection of changes" (or a group, a set, a list if  you find a problem with _collection_ wording). Basically I suggest emphasizing there's a **group** of changes, in a stronger way than just using the plural.